Gemfile

```diff
@@ -3,7 +3,6 @@
 
 source "https://rubygems.org"
 gem "logstash-core", "3.0.0.dev", :path => "./logstash-core"
-# gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
```

Comment:
bitten, bitten! man we should end up removing this commented out line (at the end of the day is not necessary :P)